ip_country/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+// These must be before the rest of the modules
+// in order to be able to use the macros.
+
+pub mod bit_queue;
+pub mod countries;
+pub mod country_block_serde;
+pub mod country_block_stream;
+pub mod country_finder;
+pub mod ip_country;
+#[rustfmt::skip]
```

Comment:
I see no use in this attribute macro.

Reply:
this annotation skips formatter for dbip_country.rs, because otherwise formatter wants to reformat that large code of IPs

Reply:
Yes. It doubles the same attributes which are inside the file, but it's okay. Ignore this considered solved.